executor/asyncloaddata/progress.go

```diff
@@ -24,14 +28,25 @@ type Progress struct {
 	// SimpleSeekerOnReadCloser on MySQL client connection which doesn't support
 	// it.
 	SourceFileSize int64
-	// LoadedFileSize is the size of the data that has been loaded in bytes.
+	// LoadedFileSizeSetter is used by multiple workers to record the progress,
+	// its value is loaded into LoadedFileSize when String().
+	LoadedFileSizeSetter atomic.Int64 `json:"-"`
```

Comment:
can merge with below one?

Reply:
Do you mean we merge `LoadedFileSizeSetter` with `LoadedFileSize`? I think I can write a customized json Marshaller/Unmarshaller to do it